hack/make-rules/image.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+###
+#Copyright 2021 The KubeEdge Authors.
+#
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+###
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBEEDGE_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd -P)"
+
+ALL_IMAGES_AND_TARGETS=(
+  #{target}:{IMAGE_NAME}:{DOCKERFILE_PATH}
+  cloudcore:cloudcore:build/cloud/Dockerfile
+  admission:admission:build/admission/Dockerfile
+  edgecore:edgecore:build/edge/Dockerfile
+  edgesite-agent:edgesite-agent:build/edgesite/agent-build.Dockerfile
+  edgesite-server:edgesite-server:build/edgesite/server-build.Dockerfile
+  csidriver:csidriver:build/csidriver/Dockerfile
+  iptablesmanager:iptables-manager:build/iptablesmanager/Dockerfile
+)
+
+GO_LDFLAGS="$(${KUBEEDGE_ROOT}/hack/make-rules/version.sh)"
+IMAGE_TAG=$(git describe --tags)
```

Comment:
Now the `IMAGE_TAG` is from `git describe --tags`, for daily build, what IMAGE_TAG should used? cc @zhu733756 @zc2638 Is the `git describe --tags` still ok?

Reply:
> Now the `IMAGE_TAG` is from `git describe --tags`, for daily build, what IMAGE_TAG should used? cc @zhu733756 @zc2638 Is the `git describe --tags` still ok?

I think it is ok for me.